src/components/Gameboard.module.css

```diff
@@ -14,3 +14,15 @@
   border-width: 3px;
   border-color: yellow;
 }
+
```

Comment:
These styles can be reworked a little bit to avoid styles repetition. We can do something like:

```
.selectedCard, .isNotPartSet, .isPartSet {
  border-style: solid;
  border-width: 3px;
}

.selectedCard {
  border-color: yellow;
}

.isPartSet {
  border-color: green;
}

.isNotPartSet {
  border-color: red;
}

```